be/src/olap/page_cache.h

```diff
@@ -85,7 +136,7 @@ class StoragePageCache {
     // This function is thread-safe, and when two clients insert two same key
     // concurrently, this function can assure that only one page is cached.
     // The in_memory page will have higher priority.
-    void insert(const CacheKey& key, const Slice& data, PageCacheHandle* handle,
+    void insert(const CacheKey& key, DataPage<>& data, PageCacheHandle* handle,
```

Comment:
why using DataPage<> ? Is DataPage ok?

Reply:
No, `DataPage` is a template class, `<>` must be specified, even if the template parameter is the default